packs/smart_items/assets/signpost_wood_short/composite.json

```diff
@@ -0,0 +1,116 @@
+{
+  "version": 1,
+  "components": [
+    {
+      "name": "core::Transform",
+      "data": {
+        "513": {
+          "json": {
+            "position": {
+              "x": 0,
+              "y": 0.9852020740509033,
+              "z": 0.31922149658203125
+            },
+            "rotation": {
+              "x": 0,
+              "y": 1,
+              "z": 0,
+              "w": 6.123234262925839e-17
+            },
+            "scale": {
+              "x": 0.2676096558570862,
+              "y": 0.2676096558570862,
+              "z": 0.2676096558570862
+            },
+            "parent": 0
+          }
+        }
+      }
+    },
+    {
+      "name": "core::Animator",
+      "data": {
+        "0": {
+          "json": {
+            "states": []
+          }
+        },
+        "513": {
+          "json": {
+            "states": []
+          }
+        }
+      }
+    },
+    {
+      "name": "core-schema::Name",
+      "data": {
+        "0": {
+          "json": {
+            "value": "Signpost Wood Short"
+          }
+        },
+        "513": {
+          "json": {
+            "value": "Text"
+          }
+        }
+      }
+    },
+    {
```

Comment:
will the text of this smart be dynamic?

Reply:
Not for now, because we don't have a "change text" action

It would be amazing to have that in the future. So I'm leaving this here for when it's possible 🙏